lib/req/steps.ex

```diff
@@ -197,17 +194,33 @@ defmodule Req.Steps do
     end
   end
 
-  def load_netrc(request, true) do
-    load_netrc(request, Path.join(System.user_home!(), ".netrc"))
+  defp load_netrc(path) do
+    case File.read(path) do
+      {:ok, ""} ->
+        raise ".netrc file is empty."
```

Comment:
```suggestion
        raise ".netrc file is empty"
```

btw, if we don't have the `{:ok, ""}` clause wouldn't we raise with the `cannot parse .netrc file` exception? If so, let's remove this clause and rely on the latter error instead.

Reply:
Without this clause `parse_netrc` just returns an empty map; it doesn't raise. We could add a clause that catches it and raises; I don't have a preference.

Reply:
Lets leave this as is then